feature-libs/customer-ticketing/components/customer-ticketing-list/customer-ticketing-list.component.html

```diff
@@ -1,13 +1,110 @@
+        <div class="col-11">
+          <div>
+            <span class="col-4 cx-ticketing-list-ticket-item-mobile">
+            {{ 'customerTicketing.ticketIdFull' | cxTranslate }}
+            </span>
+            <span class="col-8 cx-ticketing-list-ticket-item-mobile">
+            {{ ticket.id }}
+            </span>
+          </div>
+          <div>
+            <span class="col-4 cx-ticketing-list-ticket-item-mobile">
+            {{ 'customerTicketing.subject' | cxTranslate }}
+            </span>
+            <span class="col-8 cx-ticketing-list-ticket-item-mobile">
+            {{ ticket.subject }}
+            </span>
+          </div>
+          <div>
+            <span class="col-4 cx-ticketing-list-ticket-item-mobile">
+            {{ 'customerTicketing.ticketCategory' | cxTranslate }}
+            </span>
+            <span class="col-8 cx-ticketing-list-ticket-item-mobile">
+            {{ ticket.ticketCategory }}
+            </span>
+          </div>
+          <div>
+            <span class="col-4 cx-ticketing-list-ticket-item-mobile">
+            {{ 'customerTicketing.dateCreated' | cxTranslate }}
+            </span>
+            <span class="col-8 cx-ticketing-list-ticket-item-mobile">
+            {{ ticket.createdAt }}
+            </span>
+          </div>
+          <div>
+            <span class="col-4 cx-ticketing-list-ticket-item-mobile">
+            {{ 'customerTicketing.dateUpdated' | cxTranslate }}
+            </span>
+            <span class="col-8 cx-ticketing-list-ticket-item-mobile">
+            {{ ticket.modifiedAt }}
+            </span>
+          </div>
+          <div>
+            <span class="col-4 cx-ticketing-list-ticket-item-mobile">
+            {{ 'customerTicketing.status' | cxTranslate }}
+            </span>
+            <span class="col-8 cx-ticketing-list-ticket-item-mobile">
+            {{ ticket.status }}
+            </span>
+          </div>
+        </div>
+        <div class="col-1 cx-tickting-list-ticket-mobile-icon">
+          <cx-icon class="fa fa-angle-right fa-2x"></cx-icon>
+        </div>
+      </div>
+      <table class="table cx-table">
+        <thead>
+          <tr>
+            <th scope="col">{{ 'customerTicketing.ticketId' | cxTranslate }}</th>
+            <th scope="col">{{ 'customerTicketing.subject' | cxTranslate }}</th>
+            <th scope="col">{{ 'customerTicketing.ticketCategory' | cxTranslate }}</th>
+            <th scope="col">{{ 'customerTicketing.createdOn' | cxTranslate }}</th>
+            <th scope="col">{{ 'customerTicketing.changedOn' | cxTranslate }}</th>
+            <th scope="col">{{ 'customerTicketing.status' | cxTranslate }}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr *ngFor="let ticket of tickets">
+            <td>
+              <a [routerLink]="['/my-account/customer-ticketing', ticket.id]">{{ ticket.id }}</a>
+            </td>
+            <td>{{ ticket.subject }}</td>
```

Comment:
Subject is also clickable

Reply:
I will modify this when I have created the link to the ticket in the next pr